src/site/content/en/progressive-web-apps/codelab-make-installable/index.md

```diff
@@ -3,7 +3,9 @@ layout: codelab
 title: Make it installable
 authors:
   - petelepage
+  - beaufortfrancois
```

Comment:
```suggestion
```

Reply:
@beaufortfrancois the `author` field is reserved for the people who create the content or substantially update it. In the future it might also imply a commitment to maintain the content over time.